frontend/test/__support__/e2e/helpers/e2e-bi-basics-helpers.js

```diff
@@ -6,6 +8,52 @@ export function filter({ mode } = {}) {
   initiateAction("Filter", mode);
 }
 
+export function filterField(fieldName, { operator, value, placeholder } = {}) {
+  if (operator) {
+    changeOperator(getFilterField(fieldName), operator);
+  }
+
+  if (value) {
+    changeValue(getFilterField(fieldName), value, placeholder);
+  }
+
+  return getFilterField(fieldName);
+}
+
+export function filterFieldPopover(fieldName, { value, placeholder } = {}) {
+  getFilterField(fieldName).within(() => {
+    cy.findByTestId("select-button").click();
+  });
+
+  if (value) {
+    changeValue(popover(), value, placeholder);
+  }
+  return popover();
+}
+
+function getFilterField(fieldName) {
+  return cy.findByLabelText(`filter-field-${fieldName}`);
```

Comment:
nit: Maybe change this to `testid` so `aria-label` can be a human-readable (and translated) string